packages/tools/pie-icons-webc/README.md

```diff
@@ -91,78 +115,59 @@ export class MyAmazingComponent extends LitElement {
 }
 ```
 
-#### React
+### React
 
-To import from the package root:
+Each icon has a separate entrypoint for use in React applications. This uses our [`pie-wrapper-react`](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-wrapper-react) package.
 
 ```tsx
-// Please note we include /dist/ in the path only for React exports. This is due to how we have setup React exports to work with frameworks such as NextJS.
-import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-webc/dist/react";
-
-// If your app can support the exports set in the package.json, you can also import like so (exclude 'dist'):
-import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-webc/react";
+import { IconAlertTriangleLarge } from "@justeattakeaway/pie-icons-webc/dist/react/IconAlertTriangleLarge.js";
+import { IconCalendar } from "@justeattakeaway/pie-icons-webc/dist/react/IconCalendar.js";
 
 export default function App() {
   return (
     <div className="App">
-      <IconCalendar />
       <IconAlertTriangleLarge fill={PIE_ALIAS_COLOR_TOKEN} />
-    </div>
-  );
-}
-```
-To import a single icon:
-
-```tsx
-// Please note we include /dist/ in the path only for React exports. This is due to how we have setup React exports to work with frameworks such as NextJS
-import { IconCalendar } from "@justeattakeaway/pie-icons-webc/dist/react/IconCalendar";
-
-// If your app can support the exports set in the package.json, you can also import like so (exclude 'dist'):
-import { IconCalendar } from "@justeattakeaway/pie-icons-webc/react/IconCalendar";
-
-export default function App() {
-  return (
-    <div className="App">
       <IconCalendar />
     </div>
   );
 }
 ```
 
-#### Vue
+### Vue
 
-This package requires Node 18+, therefore, if you are using a lower version of Node please use our native Vue component package – [pie-icons-vue](https://www.npmjs.com/package/@justeattakeaway/pie-icons-vue).
+Your application needs to be using Node 18 or higher to use the icon web components. If you are using a lower version of Node, please use our native Vue component package, [`pie-icons-vue`](https://www.npmjs.com/package/@justeattakeaway/pie-icons-vue).
```

Comment:
I think we should remove the part about suggesting the vue icons package on lower node versions. That only works because we haven't picked up the ticket to upgrade that package yet. It isn't designed intentionally to be a fallback for lower node versions.

Reply:
That's a fair point. Do you think I should not mention the other package at all?

Reply:
Yeah I think don't mention it